test/io/ferguson/writebinaryarray.suppressif

```diff
@@ -0,0 +1,5 @@
+# suppress writebinaryarray on intel compiler
+
+CHPL_TARGET_COMPILER == cray-prgenv-intel
```

Comment:
Do you need to supressif for the no prgenv version too? i.e. also skip for `CHPL_TARGET_COMPILER == intel`